### ISSUE TITLE
Vysor can't install APK or control MIUI devices

### ISSUE THREAD

**Issue by koush** [koush/vysor.io#833]
By default, MIUI prevents installation and simulating input over USB. You must:

- ENABLE "Install via USB"
- ENABLE "USB Debugging (Security Settings)" (This is a different additional option from "USB Debugging")
- DISABLE "Verify apps over USB"

![screenshot-1625732269863](https://user-images.githubusercontent.com/73924/124887868-6d781800-df8a-11eb-802a-66bdaab7e969.png)

**Comment by princeteck** [comment]
we dont have the `install apps via usb` in vivo funtouch os 10.5 and nokia 5.3 device. where to find this settings ? @koush

**Comment by smibrandon** [comment]
Same: no `install apps via usb` on **Pixel 2** and **Pixel 4** running **Android 11**

> we dont have the `install apps via usb` in vivo funtouch os 10.5 and nokia 5.3 device. where to find this settings ? @koush

**Comment by lleandroo93** [comment]
Even enabling USB debugging, Install via USB, US debugging (Security settings) and disabling Verify apps over USB, I can't view my device Redmi 4A using Vysor Free Version 4.1.77 (Runtime 4.1.77)
I don't know if these two files generated when I clicked Bug Report would help, but I'm attaching anyway.

[45ff386f-c1bb-42bd-a995-439f719d0353.txt](https://github.com/koush/vysor.io/files/6969107/45ff386f-c1bb-42bd-a995-439f719d0353.txt)
[f2dde983-6a93-48e7-9286-8ef2a9e321ee.txt](https://github.com/koush/vysor.io/files/6969108/f2dde983-6a93-48e7-9286-8ef2a9e321ee.txt)

**Comment by PraveenKoppula** [comment]
Even I dont see `install apps via usb` option in Samsung S10 device.

> Same: no `install apps via usb` on **Pixel 2** and **Pixel 4** running **Android 11**
> 
> > we dont have the `install apps via usb` in vivo funtouch os 10.5 and nokia 5.3 device. where to find this settings ? @koush

**Comment by Sage222** [comment]
anyway to do this with a broken screen? I have TWRP and Root.

**Comment by AbbasJafariTrax** [comment]
I had the same issue but when I restarted my phone the problem was fixed. My phone is Redmi note 8 pro

**Comment by Radek37** [comment]
Hodně jsem restartoval a asi jsem udělal podle manuálu. Když klepnu na šipku- View Device, tak se odinstaluje z telefonu Vysor. Dělá to několikrát. Už nevím co mám nastavit na telefonu. Máte kontakt na firmu Vysor?

**Comment by Radek37** [comment]
I rebooted a lot and probably followed the manual. When I click on the View Device arrow, Vysor is uninstalled from the phone. He does this several times. I don't know what to set on my phone anymore. Do you have a contact for Vysor?

**Comment by marcelosanchez** [comment]
Tengo el mismo problema, aun no existe una solucion?

**Comment by Boburbekkk** [comment]
menda o'xshamayapdi

**Comment by Henry1970** [comment]
I have the same problem. 
We have more information about this?

Regards

**Comment by Sage222** [comment]
just use https://github.com/Genymobile/scrcpy

**Comment by HoundDogTech** [comment]
My Pocophone (Xiaomi owned, runs on MIUI) was running fine with Vysor, now after the most recent update on 2 different computers Vysor isn't working anymore?

I done all the above and it's still not working, am I missing something here?

**Comment by HoundDogTech** [comment]
> My Pocophone (Xiaomi owned, runs on MIUI) was running fine with Vysor, now after the most recent update on 2 different computers Vysor isn't working anymore?
> 
> I done all the above and it's still not working, am I missing something here?

I was able to do this, fix the issue, because the phone requested a sim card for me to change (select or deselect) one of the options, I forgot which of the above it was.
After adding a SIM card, restarting phone and following the above, I was able to change one of the selections. I don't typpically use a SIM card in this phone and it still works after removing the SIM. Vysor working now.
